lib/ts/config.ts

```diff
@@ -496,11 +508,29 @@ export async function getQuestions(flags: UserFlags) {
                 return "Invalid project name: " + validations.problems![0];
             },
         },
+        {
+            name: "ui",
+            type: "list",
+            message: "Choose the ui built type for your frontend:",
+            choices: mapOptionsToChoices(uiBuildOptions),
+            when: (answers: Answers) => {
+                if (flags.ui) {
+                    answers.ui = flags.ui;
```

Comment:
we should not map the flags to the answers

Reply:
The code is updated with the requested changes, hence resolving this comment.